local.settings.example.json

```diff
@@ -1,8 +1,13 @@
 {
   "IsEncrypted": false,
   "Values": {
-    "AzureWebJobsStorage": "STORAGE_CONNECTION_STRING",
-    "SendGrid_API_Key": "SENDGRID_API_KEY",
-    "FUNCTIONS_WORKER_RUNTIME": "python"
+    "FUNCTIONS_WORKER_RUNTIME": "python",
+	"AzureWebJobsStorage": "UseDevelopmentStorage=true",
+	"SendGrid_API_Key": "<SENDGRID_API_KEY>",
+	"AzureServiceBus": "<SERVICE_BUS_CONNECTION>",
```

Comment:
AzureServiceBus variable is not used in HttpTriggerSqlClient, please check if it is used on other triggers.